pkgs/development/ocaml-modules/base64/default.nix

```diff
@@ -13,9 +13,16 @@ buildDunePackage {
 
   minimumOCamlVersion = "4.03";
 
-  buildInputs = [ alcotest bos ];
+  buildInputs = [ bos ];
+
+  # Fix test-suite for alcotest ≥ 1.0
+  patches = [(fetchpatch {
+    url = "https://github.com/mirage/ocaml-base64/commit/8d334d02aa52875158fae3e2fb8fe0a5596598d0.patch";
```

Comment:
Why not update `base64` to [3.4.0](https://github.com/mirage/ocaml-base64/releases/tag/v3.4.0) which contains this commit?

Reply:
Maybe next PR…